components/layout_2020/flow/inline/text_run.rs

```diff
@@ -140,13 +142,15 @@ impl TextRunSegment {
             soft_wrap_policy = SegmentStartSoftWrapPolicy::Force;
         }
 
+        let mut byte_processed = 0;
```

Comment:
If you make this ByteIndex(0) you can add other ByteIndex values to it directly, which will allow removing some `.0` bits from this code.